.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -30,7 +30,7 @@ body:
     id: version
     attributes:
       label: Version
-      description: What was the version this was found on?
+      description: What was the version this was found on (use branch name if unreleased)?
```

Maybe we alter this to give more examples?

```suggestion
      description: |-
        What was the version this was found on?
        
        eg. a branch name, a commit id, or a version number
```